.rubocop_todo.yml

```diff
@@ -343,6 +343,7 @@ Metrics/ModuleLength:
     - 'spec/services/permissions/order_spec.rb'
     - 'spec/services/variant_units/option_value_namer_spec.rb'
     - 'spec/support/request/stripe_stubs.rb'
+    - 'spec/lib/stripe/payment_intent_validator_spec.rb'
```

Comment:
Adding it in the rubocop_todo implies we intend to fix it down the line. I think if we are happy to ignore the rule in this case I would use these in the spec file :  
``` ruby
# rubocop:disable Metrics/ModuleLength
....
# rubocop:enable Metrics/ModuleLength

Reply:
> ignore the rule in this case I would use these in the spec file

That makes sense... this way we would not loose track of this rubocop violation. In any case, it won't be necessary this time, I've following Maikel's suggestion above.